### DISCUSSION TITLE
H115i PRO XT

### DISCUSSION THREAD

**Discussion by galtline** [liquidctl/liquidctl#420] [Q&A]
Please describe the issue or request as clearly and completely as possible.

Getting The following:

$ liquidctl --match H115i set sync color fixed 0080ff
ERROR: Error: missing --unsafe features for Corsair H115i Pro XT (experimental): 'pro_xt_lighting'
ERROR: More information is provided in the corresponding device guide

**Comment by jonasmalacofilho** [comment]
On the version you have, lighting control of Pro XT devices is experimental and requires the `--unsafe pro_xt_lighting` flag on all related commands.

By the way, you may want to consider updating liquidctl: 1.5.1 is already almost a year old, and there have been some (admittedly small) improvements to the Pro XT driver since.

P.S. Will move this to the Discussions tab (Q&A section).

**Reply by galtline** [reply]
What version can I download if running Linux Mint. The graph showed 1.5.1

**Reply by jonasmalacofilho** [reply]
Manually, you can install any version you'd like.

The graph/table in the readme shows the version which is already packaged in each distribution's official repositories.

By the way, you can also ask your distribution to update the version they package.  If I remember correctly you're using a LTS version, so they would likely only update it to 1.5.2.  Still, requesting it may get them to update to 1.8.1 ([or 1.9.0](https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md#release-cycle-and-pre-release-freeze-periods)) for their next release.

**Reply by galtline** [reply]
will do Thank you
